hardhat.config.ts

```diff
@@ -90,6 +90,7 @@ export default {
   },
   networks: {
     hardhat: {
+      hardfork: "berlin",
```

Comment:
I see this [should be](https://hardhat.org/config/) the default value, is this to "freeze" the fork in case there are incompatible changes in a future fork?

Reply:
Ah, that was left over when I was trying some stuff out. I'll remove it. In fact, the block gas limit setting can also be removed as the default is the same here.